src/status_im/native_module/impl/module.cljs

```diff
@@ -63,7 +63,8 @@
                                  "JavaScriptCore"
                                  "OttoVM")
                                " jail initialized")]
-          (.initJail status init-js' #(log/debug log-message)))))))
+          (.initJail status init-js' #(do (re-frame/dispatch [:jail-initialized])
```

Comment:
we can dispatch `:initialize-app` here

Reply:
@flexsurfer good idea! Removed a lot of unused stuff, please take a look again.